lib/Runtime/Library/JavascriptLibraryBase.h

```diff
@@ -269,6 +269,7 @@ namespace Js
         Var minValue;
         Var maxValue;
         Var negativeZero;
+        Var experimentalVersion; // Wasm.experimentalVersion
```

Comment:
Put this in `JavascriptLibrary.h` changing `JavascriptLibraryBase` will break api compatibility with Edge which is unnecessary in this case